### PR TITLE
[Docs] update Japanese translation of mod_tap.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

update Japanese translation of docs/mod_tap.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

**Review by s-show — APPROVED** [review]

File: docs/ja/mod_tap.md [hunk 1]
```diff
@@ -55,11 +58,13 @@ MT(MOD_LCTL | MOD_LSFT, KC_ESC)
...
 
-さらに、Windows でリモートデスクトップ接続を使う場合に、問題が発生する場合があります。これらのコードはシフトを非常に高速に送信するため、リモートデスクトップはコードを見逃すかもしれません。
+これを拡張してもせいぜい複雑になるだけでしょう。32ビットキーコードに移行すると、これの多くが解決されますが、キーマップマトリックスが使用する領域が2倍になります。また、問題が起きる可能性もあります。タップしたキーコードにモディファイアを適用する必要がある場合は、[タップダンス](ja/feature_tap_dance.md#example-5-using-tap-dance-for-advanced-mod-tap-and-layer-tap-keys)を使うことができます。
 
-これを修正するには、リモートデスクトップ接続を開き、「オプションの表示」を開き、「ローカル リソース」タブを開きます。キーボードセクションで、ドロップダウンを「このコンピューター」に変更します。これにより問題が修正され、キャラクタが正しく動作するようになります。
+さらに、Windows でリモートデスクトップ接続を使う場合に、問題が発生する場合があります。これらのキーコードは人よりも速くキーイベントを送信するため、リモートデスクトップはコードを見逃すかもしれません。
+これを修正するには、リモートデスクトップ接続を開き、「オプションの表示」を開き、「ローカル リソース」タブを開きます。キーボードセクションで、ドロップダウンを「このコンピューター」に変更します。これにより問題が修正され、キャラクタが正しく動作するようになります。これにより問題が修正され、キャラクタが正しく動作するようになります。
+[`TAP_CODE_DELAY`](ja/config_options.md#behaviors-that-can-be-configured) を増やすことで緩和することもできます。
```
[the remark this answers is not in the document]
Comment (umi-umi) [reply docs/ja/mod_tap.md:67]: ご指摘ありがとうございます。
ja/config_options.mdのほうも更新しました。

**Review by shelaf — APPROVED** [review]

**Review by mtei — APPROVED** [review]

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks!

**Pull request merged**